mvp.feature

```diff
@@ -1,140 +1,140 @@
 # encoding: utf-8
 # language: en
 
-Feature: User with no Author identity in the Authority Register for Personas sees possible Author identities
+Feature: MVP
   # This assumes that the Authority Register for Personas is used as an identity keyring
-  Scenario:
+  Scenario: 1. User with no Author identity in the Authority Register for Personas sees possible Author identities
     Given that a user has a valid Feide ID and password
     And they have do not have an identity in the Authority Register for Personas
     When they log in to NVA
     Then the user sees a list of possible Author identities based on a free-text search for their registered name from Feide in the Authority Register for Personas
     And they see the last registered publication for each Author identity based on a SCN (System Control Number) lookup in Alma
 
-  Scenario: User creates an Author identity
+  Scenario: 2. User creates an Author identity
     Given that a user with no Author identity in the Authority Register for Personas sees possible Author identities
     When they select a Author identity
     Then the user's Feide ID is added to the record for the Author identity in the Authority Register for Personas
     And the user's Feide organistation id is mapped to a Organisational ID (Cristin ID)
```

Comment:
So a person can have a different organisation ids in context of Feide and Cristin?